content/docs/2.0/scalers/ibm-mq.md

```diff
@@ -0,0 +1,96 @@
+> © Copyright IBM Corporation 2020
+> 
+> Licensed under the Apache License, Version 2.0 (the "License"); you
+> may not use this file except in compliance with the License. You may
+> obtain a copy of the License at
+> 
+> http://www.apache.org/licenses/LICENSE-2.0
+> 
+> Unless required by applicable law or agreed to in writing, software
+> distributed under the License is distributed on an "AS IS" BASIS,
+> WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+> implied. See the License for the specific language governing
+> permissions and limitations under the License.
+
+
++++
+title = "IBM MQ"
+layout = "scaler"
+availability = "v2.0+"
+maintainer = "Community"
+description = "Scale applications based on IBM MQ Queue"
+go_file = "ibmmq_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `ibmmq` trigger for IBM MQ Queue.
+
+```yaml
+triggers:
+    - type: ibmmq
+      metadata:
+        queueLength: '5' # OPTIONAL - Queue length target for HPA. Default: 5 messages
+        host: <ibm-host> # REQUIRED - IBM MQ Queue Manager Admin REST Endpoint
+        queueName: <queue-name> # REQUIRED - Queue Name
```

Comment:
Nit, moving optionals to the end
```suggestion
        host: <ibm-host> # REQUIRED - IBM MQ Queue Manager Admin REST Endpoint
        queueName: <queue-name> # REQUIRED - Queue Name
        queueLength: '5' # OPTIONAL - Queue length target for HPA. Default: 5 messages
```

Reply:
Hi @tomkerkhove, optional parameters have been moved to the end